src/components/TreeFilter/VirtualizedTreeView.tsx

```diff
@@ -642,6 +642,7 @@ export class VirtualizedTreeView extends React.PureComponent<IVirtualizedTreeVie
             this.props.onValuesSelected(this.props.filterId, { type: FilterSelectionEnum.None, selectedIDs: [] });
         }
         this.setState(prevState => ({ ...prevState, partiallyCheckedItemIds: newPartiallyChecked }));
+        this._list.forceUpdateGrid();

```

Comment:
how come this state change didn't trigger an update with the regular lifecycle?

Reply:
state only stores info on partially checked list. if none/all of it is selected, state doesn't change.